### PR TITLE
First guide for the parent page of AI and machine learning

### PR DESCRIPTION
**Pull request by baaastijn**
Comparative matrix about AI Tools

New Public Cloud category added (AI and machine learning) to follow new documentation guidelines

### CODE REVIEW

File: pages/platform/ai/apps_01_tokens/guide.en-gb.md [hunk 1]
```diff
@@ -0,0 +1,151 @@
...
+
+### Using a token to query an AI App
+
+With the token we generated in the previous step we will now query the AI App. For this demonstration, we deployed a simple Hello World app that always responds `Hello, World!`.
+
+In the details of our AI App above the Tags you can get the access URL of your AI App.
+
+#### Browser
+
+If access the AI App access URL from your browser you reach a Login page:
```
Comment (OVH-Team-Guides) [inline pages/platform/ai/apps_01_tokens/guide.en-gb.md]: @christophe-rannou, I do not understand that sentence. Can you maybe rephrase it please ?

File: pages/platform/ai/apps_01_tokens/guide.en-gb.md [hunk 2]
```diff
@@ -0,0 +1,151 @@
...
+
+![token generation input operator](images/07-token-gen-input-op.png){.thumbnail}
+
+Additional information about the use of a token to manage an AI Training resource can be found [here](https://docs.ovh.com/gb/en/ai-training/app-token-cli/#use-the-app-token).
+
+### Using a token to query an AI App
+
+With the token we generated in the previous step we will now query the AI App. For this demonstration, we deployed a simple Hello World app that always responds `Hello, World!`.
+
+In the details of our AI App above the Tags you can get the access URL of your AI App.
```
Comment (OVH-Team-Guides) [inline pages/platform/ai/apps_01_tokens/guide.en-gb.md]: @christophe-rannou,  I do not understand that sentence. Can you maybe rephrase it please ?

Comment (christophe-rannou) [reply pages/platform/ai/apps_01_tokens/guide.en-gb.md]: refers to the same details as in L37

Comment (OVH-Team-Guides) [reply pages/platform/ai/apps_01_tokens/guide.en-gb.md]: proposition: "You can get the access URL of your AI App in the details of the AI App, under **Tags**."

File: pages/platform/ai/apps_01_tokens/guide.en-gb.md (hunk 1 above)
Comment (OVH-Team-Guides) [reply pages/platform/ai/apps_01_tokens/guide.en-gb.md]: proposition: "When accessing the AI App via its URL in your browser, you will reach a Login page:"

**Comment by OVH-Team-Guides** [comment]
Thanks for your contribution @baaastijn and @christophe-rannou
